make-dist

```diff
@@ -35,7 +35,7 @@ echo "version $version"
 # update submodules
 echo "updating submodules..."
 force=$(if git submodule usage 2>&1 | grep --quiet 'update.*--force'; then echo --force ; fi)
-if ! git submodule sync || ! git submodule update $force --init --recursive; then
+if ! git submodule sync || ! git submodule update $force --init --recursive --progress; then
```

Comment:
My understanding is that this script is run only by automated tooling. I don't think the `--progress` flag is useful in that context?

Reply:
Won't this be useful when automated tooling gets stuck because download is too slow? And having this won't harm only add more info to the log/output; it won't harm in any way.

Reply:
I guess? I'm fine either way.